Innostax/tavajs
# 182/Integrate Sentry Logger Service.

Resolves: #182

Description:

Steps to test Sentry integration:-
1.Create an account on [Sentry.io](http://sentry.io/) and generate a DSN url.
2.Paste the DSN url in sentry.js file.
3.Run app and make a GET request on the users/logger/test.
4. Check the sentry dashboard, a log will be created.
